### PR TITLE
Adds missing necromancer hood to magivend

### LINKED ISSUE #10313: magivend missing things
(WEB REPORT BY: pvak REMOTE: 172.93.109.202:7777)

> Revision (Should be above if you're viewing this from ingame!)
> 
> General description of the issue
> the magivend is missing the necromancer hat
> 
> What you expected to happen
> to have the hat
> 
> What actually happened
> no hat
> 
> Steps to reproduce if possible
> get a magivend
> find out it doesnt have the necromancer hat

### PR DESCRIPTION
**Pull request by Konater**
fixes #10313

### CODE REVIEW

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -1605,6 +1606,7 @@ var/global/num_vending_terminals = 1
 		/obj/item/clothing/shoes/sandal/marisa = 5,
 		/obj/item/weapon/staff = 5,
 		/obj/item/weapon/staff/broom = 5,
+		/obj/item/clothing/glasses/monocle = 5,
```
Comment (ComicIronic) [inline code/game/machinery/vending.dm:1609]: You lied about this PR in the title

Comment (Konater) [inline code/game/machinery/vending.dm:1609]: But I did add it

Comment (Imagebard) [inline code/game/machinery/vending.dm]: Its just some monocles who ares

Comment (ComicIronic) [inline code/game/machinery/vending.dm:1609]: I don't like non-atomic PRs. If it's an issue PR, it's an issue PR.

Comment (Imagebard) [inline code/game/machinery/vending.dm]: Oh christ who gives a fuck

Comment (ComicIronic) [inline code/game/machinery/vending.dm:1609]: Me, obviously.
Bit of a silly question.

**Pull request merged**